src/main/java/io/codelex/arrays/exercises/Hangman.java

```diff
@@ -0,0 +1,119 @@
+package io.codelex.arrays.exercises;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Hangman {
+
+        private static final String [] wordBank =
+                {"java", "cat", "yesterday", "indiana", "makeup", "boots",
+                        "programming", "polish", "transportation", "breakfast", "strawberry", "cake",
+                        "remote", "cookies", "terminology", "television", "cranberry", "tool",
+                        "caterpillar", "juice", "watermelon", "laptop", "toe", "toad",
+                        "aardvark", "capitol", "kitten", "sanguine", "meow", "burger"};
+        private final String chosenWord;
+        private final ArrayList<Character> rightGuesses;
+        private final ArrayList<Character> wrongGuesses;
+        private final Scanner userIn = new Scanner(System.in);
+
+
+	public Hangman() {
```

Comment:
šis varētu būt package-private un pati klase arī, bet varbūt par to vēl nav runāts līdz šī commit taisīšanas :smile: